### ISSUE TITLE
Support models with offset term

### ISSUE THREAD

**Issue by Marchen** [Marchen/cv.models#24]
`redict.gbm` produces 
```
Warning message:
In predict.gbm(model.gbm, newdata = d, type = "response", n.trees = model.gbm$n.trees) :
  predict.gbm does not add the offset to the predicted values.
```

So when model having offset term, performance measures may not be correct.

Because GLM support prediction with offset term, model specific handling is requied.
So I think I need to update `model.adapter`.

**Comment by Marchen** [comment]
We need to implement a method like "has.offset" in `model.adapter`.

**Comment by Marchen** [comment]
Fixed in 0.1.4.
